modules/julia/gen/funclist.csv

```diff
@@ -0,0 +1,666 @@
+cv.borderInterpolate
+cv.copyMakeBorder
+cv.copyMakeBorder
+cv.add
+cv.add
```

Comment:
Why are they duplicate?

Reply:
Happened because of Mat and UMat entires. Not needed, will fix